src/crimson/os/seastore/transaction.h

```diff
@@ -191,11 +207,7 @@ class Transaction {
   friend class Cache;
   friend Ref make_test_transaction();
 
-  /**
-   * If set, *this may not be used to perform writes and will not provide
-   * consistentency allowing operations using to avoid maintaining a read_set.
-   */
-  const bool weak;
+  const src_t src;
```

Comment:
nit, might want to rearrange the layout of this struct for smaller memory footprint.

Reply:
moved to the last